src/azure-cli-core/azure/cli/core/breaking_change.py

```diff
@@ -250,8 +250,10 @@ def register(self, cli_ctx):
         elif self.is_command_group(cli_ctx):
             command_group = cli_ctx.invocation.commands_loader.command_group_table[self.command_name]
             if not command_group:
-                self._register_to_direct_sub_cg_or_command(cli_ctx, self.command_name, self.to_tag(cli_ctx))
-            else:
+                loader = self.find_suitable_command_loader(cli_ctx)
```

Comment:
Will multiple modules founded for one command group?

Reply:
If multiple modules are found for one command group, the method `find_suitable_command_loader` will always choose the first one to create the command group. This is because our goal is simply to create a command group object. The command groups created by different command loaders through `CommandLoader.command_group` are almost identical.